docker-compose.dev.yml

```diff
@@ -1,5 +1,25 @@
 version: '3.3'
+
+# This file is intended to overwrite certain settings from `docker-compose.yml`.
+# These changes make it easier to develop,
+# with a tighter feedback loop on changes
+# and some additional debugging tools installed.
+# To activate development mode as defined by this file,
+# add this line to your `.env` file:
+#
+# COMPOSE_FILE=docker-compose.yml:docker-compose.dev.yml
+
 services:
+  client:
+    volumes:
+      # Be sure you have a local copy of the `tb-mobile-app` repo,
+      # and reference it with the `TB_MOBILE_CONTEXT` variable in `.env`.
+      - ${TB_MOBILE_CONTEXT}:/usr/src
```

Comment:
Will this still work if `TB_MOBILE_CONTEXT` is a link to Github ([as is the default](https://github.com/uwcirg/tb-api/tree/develop/docker-compose.yml#L10))?

If not, can you use a different environment variable?

Reply:
No, it'll only work if that points to a local directory.

How about I comment out that line by default, so people can opt-in if they're working with a local copy?